GetRelayConfiguration/Program.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Communication;
+using Azure.Communication.Identity;
+using Azure.Communication.NetworkTraversal;
+using Azure.Communication.NetworkTraversal.Models;
+using Microsoft.MixedReality.WebRTC;
+
+namespace GetRelayConfiguration
+{
+    class Program
+    {
+        private static CommunicationRelayClient _communicationRelayClient;
+        private static CommunicationIdentityClient _communicationIdentityClient;
+
+        static async Task Main(string[] args)

```

Comment:
What does this do?

Reply:
This is not for incoming routing, but its changes are due to taking the changes from main branch in sync. This was written for getting underline servers.